test/collectionUtils/AnyResourceTransformer.test.ts

```diff
@@ -0,0 +1,93 @@
+import AnyResourceTypeTransformer from "../../src/collectionUtils/AnyResourceTypeTransformer";
+import EntityDefinitionBuilder from "../model/dataTypes/EntityDefinitionBuilder";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+import MemberVariable from "../../src/model/dataTypes/MemberVariable";
+import DataType from "../../src/model/dataTypes/DataType";
+import EntityImports from "../../src/model/dataTypes/EntityImports";
+import FilePath from "../../src/model/dataTypes/FilePath";
+
+describe("AnyResourceTypeTransformer", () => {
+  const baseDir = FilePath.getInstance("/tmp");
+  const transformer = new AnyResourceTypeTransformer(baseDir);
+  let regularEntity: EntityDefinition;
+  let resourceMemberEntity: EntityDefinition;
+  let resourceChoiceEntity: EntityDefinition;
+
+  beforeEach(() => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)